shared/lib_utility_rate_equations.cpp

```diff
@@ -242,11 +279,11 @@ void rate_data::init_energy_rates(bool gen_only) {
 				{
 					for (size_t tier = 0; tier < num_tiers; tier++)
 					{
-						br.at(period, tier) = m_month[m].ec_tou_br_init.at(period, start_tier + tier);
-						sr.at(period, tier) = m_month[m].ec_tou_sr_init.at(period, start_tier + tier);
-						ub.at(period, tier) = m_month[m].ec_tou_ub_init.at(period, start_tier + tier);
+						br.at(period, tier) = m_month[m].ec_tou_br_init.at(period, tier_numbers[tier]);
```

Comment:
Great - another level of abstraction to handle the irregular numbering in the URDB!